react/components/Tooltip/TooltipPopup.tsx

```diff
@@ -1,6 +1,10 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable max-params */
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Does it exist another issue to resolve this error?

Reply:
Not yet, I'll create it after merges (like the lint issues one).